frontend/Makefile.help

```diff
@@ -106,12 +106,7 @@ clobber-frontend: clean-frontend-docs FORCE
 
 test-frontend: $(LIBCOMPILER_BUILD_DIR) FORCE
 	@echo "Making and running the frontend tests..."
-	@if [ -f $(LIBCOMPILER_BUILD_DIR)/Makefile ]; then \
-	  JOBSFLAG=`echo "$$MAKEFLAGS" | sed -n 's/.*\(-j\|--jobs=\) *\([0-9][0-9]*\).*/-j\2/p'` ; \
-	  cd $(LIBCOMPILER_BUILD_DIR) && $(MAKE) tests && cd frontend/test && ctest $$JOBSFLAG . ; \
-	else \
-	  cd $(LIBCOMPILER_BUILD_DIR) && $(CMAKE) --build .  --target tests && cd frontend/test && ctest . ; \
-	fi
+	@cd $(LIBCOMPILER_BUILD_DIR) && $(CMAKE) --build .  --target tests && cd frontend/test && ctest .
```

Comment:
It looks like before we were making sure not to do a parallel make - I'm assuming that we don't have to worry about firing off too many parallel jobs now because we're invoking CMake directly?